django/http/multipartparser.py

```diff
@@ -740,4 +756,4 @@ def __iter__(self):
         boundarystream = InterBoundaryIter(self._stream, self._separator)
         for sub_stream in boundarystream:
             # Iterate over each part
-            yield parse_boundary_stream(sub_stream, 1024)
+            yield parse_boundary_stream(sub_stream)
```

Comment:
Is it not enough to pass `MAX_TOTAL_HEADER_SIZE`?
```suggestion
            yield parse_boundary_stream(sub_stream, MAX_TOTAL_HEADER_SIZE)
```
I'd like to avoid API changes (I'm aware it's an undocumented private API.)

Reply:
`MultiPartParser` instance is created behind the scenes when parsing requests, a module constant will allow to change the current behavior of Django internals.

Reply:
Yes, I can do that but it still feels as a hack. Of course I have the option to reimplement the whole class but it just seems a bit wasteful to fork.

Reply:
I'm not sure we need this :shrug: . `MAX_TOTAL_HEADER_SIZE` cannot be change accidentally, so folks who need to do this can optimize it's value for their specific use case.